src/components/test/code-snippet.jsx

```diff
@@ -43,20 +43,22 @@ class CodeSnippet extends Component {
 
   render() {
     const { className, code, lang, highlight, label, showLabel } = this.props;
-    let codeHtml = code;
-
-    // Add - expected + actual to top of diffs
-    if (lang === 'diff') {
-      const expected = `<span class='${cx('code-diff-expected')}'>+ expected</span>`;
-      const actual = `<span class='${cx('code-diff-actual')}'>- actual</span>`;
-      codeHtml = `${expected}&nbsp;&nbsp;${actual}\n\n${code}`;
-    }
-
     const cxName = cx(className, lang, { hljs: !highlight });
 
     return !!code && (
       <pre className={ cxName } ref={ node => (this.node = node) }>
-        <code dangerouslySetInnerHTML={ { __html: codeHtml } } />
+        <code>
+          { lang === 'diff' &&
```

Comment:
I think this can be cleaned up a bit with some CSS.
```jsx
<code>
  { lang === 'diff' && 
    <div className={ cx('code-diff-header') }>
      <span className={ cx('code-diff-expected') }>+ expected</span>
      <span className={ cx('code-diff-actual') }>- actual</span>
    </div>
  }
  { code }
</code>
```

We can use styles to add the space between `expected` and `actual` and then to add the space below the diff header.